modules/react/tokens/lib/type/fontFamilies.ts

```diff
@@ -1,9 +1,10 @@
-import get from 'lodash/get';
+// prettier-ignore
+//@ts-ignore
+const canvasObject = typeof window !== 'undefined' && window.workday && window.workday.canvas && window.workday.canvas;
```

Comment:
this is now done 3 times in CK codebase would it be worth having a function?

Reply:
Ignore what I said, sometimes we read canvas sometimes we want theme from canvas